src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor

```diff
@@ -0,0 +1,12 @@
+﻿
+<div class="main">
+    <div class="cloud-1"></div>
+    <div class="cloud-2"></div>
+    <div class="cloud-3"></div>
+    <div>
+        <div class="title"></div>
+        <div class="description">Page Not Found</div>
+    </div>
+    <div class="disconnected-cable"></div>
+    <a href="#" Class="back-btn">Go Back</a>
```

Comment:
I think it is better to use BitLink and set href to the website's root. What do you think? @msynk

Reply:
It's better to use the `BitButton` based on the design I saw.
and also add the functionality to go back to the home page.
@sajadkardel